src/main/java/com/example/demo/feed/domain/Feed.java

```diff
@@ -21,32 +21,31 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-
-
     @Embedded
     private Writer writer;
 
     private String content;
 
     private Long likeNum;
 
+    private LocalDateTime wrtTime;
+
+    @Column(name = "is_delete")
+    private Boolean isDeleted;
```

Comment:
```suggestion
    private Boolean deleted;
```
is~ 의 내용은 메소드명에 적합한 이름입니다. 변수명에 직접 사용은 적절치 못합니다.
특히 롬복의 `@Getter`와 조합된 경우 `getIsDeleted()` 같은 이상한 메소드명이 튀어나올 수 있습니다.